src/ra_server.erl

```diff
@@ -1261,11 +1261,23 @@ handle_follower(election_timeout,
     {follower, State, []};
 handle_follower(election_timeout, State) ->
     call_for_election(pre_vote, State);
+handle_follower(try_become_leader,
+                #{cfg := #cfg{log_id = LogId},
+                  membership := Membership} = State) when Membership =/= voter ->
+    ?DEBUG("~s: follower ignored try_become_leader, non-voter: ~p",
```

Comment:
"non-voter" should probably be "replicate membership state"

Reply:
Fixed in https://github.com/rabbitmq/ra/pull/435/commits/cc30411d99e8ba120ab61d66c6918704584b7361. Thank you for your review. 
(BTW,  I have left untouched existing log messages in `ra_server.erl` that use "non-voter: ~p" to avoid increasing the diff of this PR.)